packages/native/src/useScrollToTop.tsx

```diff
@@ -101,10 +106,10 @@ export default function useScrollToTop(
           });
         }
       );
-    })
+    });
 
     return () => {
-        unsubscribers.map(unsubscribe => unsubscribe())
+      unsubscribers.map((unsubscribe) => unsubscribe());
```

Comment:
`forEach` is more appropriate since we aren't mapping the array to another array
```suggestion
      unsubscribers.forEach(unsubscribe => unsubscribe());
```